src/app/shared/mappers/wiki/wiki.mapper.ts

```diff
@@ -47,7 +48,7 @@ export class WikiMapper {
     return {
       id: response.id,
       createdAt: response.attributes.date_created,
-      createdBy: response.embeds.user.data.attributes.full_name,
+      createdBy: UserMapper.getUserInfo(response.embeds.user)?.fullName || 'Unknown author',
```

Comment:
Should this be a translation string? We should have one already for this kind of situation.